api/user.go

```diff
@@ -423,13 +423,13 @@ func GetUserSourceRepos(c *gin.Context) {
 			// add repos to list of database org repos
 			dbRepos = append(dbRepos, dbReposPart...)
 
-			// making an assumption that 50 means there is another page
+			// assume another page exists if under 100 results are returned
```

Comment:
```suggestion
			// assume this is the last page if under 100 results are returned
```
I think we want this comment changed to something like this ^ ?